3rdpartylibrary/google-maps-android-cmgn/src/main/java/com/google/maps/android/kml/KmlFeatureParser.java

```diff
@@ -1,6 +1,6 @@
 package com.google.maps.android.kml;
 
-import android.webkit.URLUtil;
+import android.webkit.*;
```

Comment:
This may not be necessary anymore, going to go back in and check. Android studio was having an issue resolving this import for me.